mypy.ini

```diff
@@ -19,6 +19,8 @@ warn_no_return = False
 warn_return_any = True
 warn_unreachable = True
 
+warn_unused_ignores = True
```

Comment:
Thanks, can you remove the line above where it's set to `False` as well? (Not sure which applies if set twice.)

Reply:
done